zadanie191.html

```diff
@@ -0,0 +1,133 @@
+<!DOCTYPE html>
+<html lang="pl">
+    <head>
+        <title>Formularz rejestracji</title>
+        <meta charset="UTF-8"> 
+        <meta name="description" content="Przykładowy formularz rejestracji ">
+        <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" 
+        rel="stylesheet" 
+        integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" 
+        crossorigin="anonymous">
+       
+    </head>
+    <body>
+      <div class="container">
+        <h1>Rejestracja</h1>
+        <form class="row g-3 needs-validation" novalidate>
+            <div class="col-md-4">
+              <label for="validationCustom01" class="form-label">Imię</label>
+              <input type="text" class="form-control" id="validationCustom01" placeholder="Wpisz imię" required>
+              <div class="valid-feedback">
+                Looks good!
+              </div>
+            </div>
+            <div class="col-md-4">
+              <label for="validationCustom02" class="form-label">Nazwisko</label>
+              <input type="text" class="form-control" id="validationCustom02" placeholder="Wpisz nazwisko" required>
+              <div class="valid-feedback">
+                Looks good!
+              </div>
```

Comment:
Widzę, że w kilku miejscach masz elementy związane z walidacją, ale w żaden sposób nie jestem w stanie sprawić, aby się one pokazały. Dodatkowo jeżeli ogólnie używasz tekstów w języku polskim, to tutaj też powinny takie być.
Zgaduję, że przy rozwiązaniu wzorowałeś się na jakimś gotowym kodzie, w czym nie ma nic złego, tylko był tam pewnie jeszcze jakiś dodatkowy javascript, który odpowiadał za pokazywanie, albo ukrywanie tych komunikatów.
Albo trzeba więc tutaj coś dodać, albo te element od walidacji usunąć

Reply:
Skorzystałem z biblioteki Bootstrap, zasugerowałem się założeniami z zadania, trochę ten kod był dla mnie mało czytelny i zrobiłem trochę na zasadzie prób i błędów.